### PR TITLE
graphiql: update readme

### LINKED ISSUE #4032: Fresh install (scaffolded app) with plugin-graphiql is not creating graphiql route
## Expected Behavior
I expect, based on the readme, that after yarn add of the plugin, and adding to plugins.ts, a /graphiql route should be created

## Current Behavior
I am getting a 404 when navigating to http://localhost:3000/graphiql

## Possible Solution
I'm not sure if there's something simple I'm missing that needs to be added to the readme(?) Or if there's an error, a helpful error message would be good.

## Steps to Reproduce

1. `npx @backstage/create-app` (follow instructions and have backstage now available at localhost:3000)
2. cd to packages/app and `yarn add @backstage/plugin-graphiql`
3. add `export { plugin as GraphiQL } from '@backstage/plugin-graphiql';` to packages/app/src/plugins.ts
4. `yarn start` back and front end and navigate to localhost:3000/graphiql

## Context
I'm fresh new to the backstage project and testing out using it for a graphql interface going for my team

## Your Environment

- NodeJS Version (v12):  14.15.4
- Operating System and Version (e.g. Ubuntu 14.04): MacOS Catalina 10.15.7
- Browser Information: Firefox

### PR DESCRIPTION
**Pull request by freben**
Fixes #4032

### CODE REVIEW

File: plugins/graphiql/README.md [hunk 1]
```diff
@@ -12,53 +12,64 @@ By exposing GraphiQL as a plugin instead of a standalone app, it's possible to p
...
-Then add an entry to your App's `plugins.ts` to import the plugin.
-
-The plugin registers a `/graphiql` route, which you can link to from the Sidebar if desired.
+```diff
+# in packages/app/src/plugins.ts
++export { plugin as GraphiQL } from '@backstage/plugin-graphiql';
+```
 
-### Adding GraphQL endpoints
+```diff
```
Comment (adamdmharvey) [inline plugins/graphiql/README.md:24]: I have never used (seen) this definition before, and now I believe I will use it a dozen times this month 🤩 🎉 thx for that

Comment (freben) [reply plugins/graphiql/README.md:24]: It's kinda neat isn't it!

**Review by jhaals — APPROVED** [review]

File: plugins/graphiql/README.md [hunk 2]
```diff
@@ -12,53 +12,64 @@ By exposing GraphiQL as a plugin instead of a standalone app, it's possible to p
...
-        body: JSON.stringify(params),
-      }).then(res => res.json());
-    },
-  },
-]);
+If all you need is a static list of endpoints, the plugin exports a `GraphQLEndpoints` class that implements the `GraphQLBrowseApi` for you. Here's an example of how you could expose two GraphQL endpoints in your App:
 
-// ApiRegistry builder created somewhere in your App
-const builder = ApiRegistry.builder();
+```diff
```
Comment (Rugvip) [inline plugins/graphiql/README.md:39]: Isn't it a bit nicer to keep this as `ts`, so we get syntax highlighting?

**Review by Rugvip — APPROVED** [review]
Without a changeset the README update won't be visible on NPM though

File: plugins/graphiql/README.md (hunk 2 above)
Comment (freben) [reply plugins/graphiql/README.md:39]: Specifically in this file, I felt that any drawback of missing highlighting was outweighed by the benefits of having it on diff form - to be able to see surrounding code and the precise lines to insert, where - without having to write a lot of free form text "in the x method of the y class under the z" etc :)

**Comment by freben** [comment]
Aww yeah good point on the readme on npm, added a changeset

**Pull request merged**